data/json/professions.json

```diff
@@ -5138,5 +5138,145 @@
                 "panties"
             ]
         }
+    },
+            {
+        "type": "profession",
+        "ident": "escape_con",
+        "name": "Escaped Prisioner",
+        "description": "When the cataclysm started you took your chance, now you roam looking for a place to stay. Hopefully those days workin in prison pay off..",
+        "points": 2,
```

Comment:
If you want to have this in core and not in "More Professions" mod, you'll need to balance it against other professions, justify its cost.
At least relative justification, like "class x has cost 2 and it also has this and that".

Reply:
-2, it's mainly supposed to be more of a challange

Reply:
-2 is what broken cyborg gets for a giant set of horrible bionics.